ipinfo-express.d.ts

```diff
@@ -0,0 +1,26 @@
+declare module "ipinfo-express" {
+    import { RequestHandler } from "express";
+
+    interface IPinfoMiddlewareOptions {
+        token?: string;
+        cache?: any;
+        timeout?: number;
+        filter?: (ip: string) => boolean;
+    }
+
+    interface IPinfoResponse {
+        ip: string;
+        hostname: string;
+        city: string;
+        region: string;
+        country: string;
+        loc: string;
+        org: string;
+        postal: string;
+        timezone: string;
```

Comment:
Sorry one more thing - this actually is a much bigger object if we use a higher-permissions token. @rm-Umar can you point @Samuel-Gill to the Nodejs SDK where this exists already? @Samuel-Gill do take a look there too.

Reply:
@Samuel-Gill check this out https://github.com/ipinfo/node/blob/master/src/common.ts

Reply:
Updated.